global.css

```diff
@@ -0,0 +1,544 @@
+
+@import url('css/bootstrap.css');
+
+/* @import url('desktop.css'); */
+
```

Comment:
- [ ] Kindly remove all unused or unwanted codes from your file. This helps to keep your codebase clean. 👍